### PR TITLE
Add PAYRUE space

### PR DESCRIPTION
**Pull request by AbstractBreazy**
includes BSC token

### CODE REVIEW

File: spaces/payrue/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "key": "payrue",
+  "symbol": "PAYRUE",
+  "name": "PayRue",
+  "defaultView": "core",
+  "address": "0xa9E81167b02cC4a79E100e3e0AF8f6Cd97Cc2727",
+  "token": "0xa9E81167b02cC4a79E100e3e0AF8f6Cd97Cc2727",
```
Comment (e-asia) [inline spaces/payrue/index.json:7]: There seem to be an issue with your the token, the interface doesn't seem compatible with it

Comment (AbstractBreazy) [reply spaces/payrue/index.json:7]: thats metamask wallet, I have not attached any tokens to it

Comment (AbstractBreazy) [reply spaces/payrue/index.json:7]: I entered the correct eth address in metamask, what could be the problem?

Comment (e-asia) [reply spaces/payrue/index.json:7]: Not sure what's wrong exactly with the token, i told my colleague to look into it he will get back to you.

File: spaces/payrue/index.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "key": "payrue",
+  "symbol": "PAYRUE",
+  "name": "PayRue",
+  "defaultView": "core",
+  "address": "0xa9E81167b02cC4a79E100e3e0AF8f6Cd97Cc2727",
```
Comment (bonustrack) [inline spaces/payrue/index.json:6]: The address doesn't seem to be a contract, it must be an ERC20 to work on the interface.

Comment (AbstractBreazy) [reply spaces/payrue/index.json:6]: if Binance Smart Chain contract/address would work since its compatible with MetaMask as well?

Comment (bonustrack) [reply spaces/payrue/index.json:6]: Yes it should. Does your token have a decimals function? If not you will need some custom setup, please join our Discord to get more info https://discord.snapshot.page

File: spaces/payrue/index.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "payrue",
+  "name": "PayRue",
+  "chainId": 56,
+  "decimals": 18,
+  "symbol": "PAYRUE",
+  "defaultView": "core",
+  "address": "0x9B44Df3318972bE845d83f961735609137C4C23c",
+  "token": "0x9B44Df3318972bE845d83f961735609137C4C23c",
```
Comment (e-asia) [inline spaces/payrue/index.json:9]: Something is wrong with the token, the call to query decimals() is failing. Is this a standard ERC20 token?

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/payrue/index.json (hunk 3 above)
Comment (AbstractBreazy) [reply spaces/payrue/index.json:9]: Sorry about that, I have fixed my mistake

File: spaces/payrue/index.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "payrue",
+  "name": "PayRue",
+  "chainId": 56,
+  "decimals": 18,
+  "symbol": "Propel",
+  "defaultView": "core",
```
Comment (bonustrack) [inline spaces/payrue/index.json:7]:
```suggestion
  "defaultView": "all",
```

**Pull request merged**